KeePassRPC/KeePassRPCExt.cs

```diff
@@ -325,7 +341,40 @@ public override bool Initialize(IPluginHost host)
 			return true; // Initialization successful
 		}
 
-        //Assembly CurrentDomain_AssemblyResolve(object sender, ResolveEventArgs args)
+        internal void AddLastAccessedEntry(Entry entry)
+        {
+            _lastAccessedEntries.Add(new LastAccessedEntry() { AccessedAt = DateTime.Now, Title = entry.Title });
+            _lastAccessedEntriesNotificationTimer.Enabled = true;
+        }
+
+	    private void LastAccessedEntriesNotificationShow()
+	    {
+            var notificationText = new StringBuilder();
+            foreach (var entry in _lastAccessedEntries)
+                notificationText.AppendFormat("{0:T}: {1}\n", entry.AccessedAt, entry.Title);
+	        var notificationString = notificationText.ToString();
+
+            if (!notificationString.Equals(_lastAccessedEntriesNotificationString))
+                _host.MainWindow.MainNotifyIcon.ShowBalloonTip(_lastAccessedEntriesNotificationBalloonTimeout, "KeePassRPC", notificationString, ToolTipIcon.None);

```

Comment:
I have written some addons that replace the tray icon on Linux with a native one. So, you should check for MainWindow.MainNotifyIcon == null here, otherwise it will crash if both plugins are installed.